landoapi/storage.py

```diff
@@ -1,16 +1,35 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from typing import TYPE_CHECKING
+
 from flask_sqlalchemy import SQLAlchemy
 from sqlalchemy.exc import DBAPIError, SQLAlchemyError
 from flask_migrate import Migrate
 
 from landoapi.systems import Subsystem
 
+if TYPE_CHECKING:
+    from landoapi.models.base import Base
+
 db = SQLAlchemy()
 migrate = Migrate()
 
 
+def _lock_table_for(
```

Comment:
Since this is something we intend to be used outside this module we should rename it to remove the `_` prefix, ie `lock_table_for`.

Out of scope for this change: maybe this function makes more sense as a method on `Base`? Then we could just do `Base.lock_table()` for example.